core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -16,20 +15,21 @@ import org.broadinstitute.dsde.workbench.model.google.{GcsBucketName, GoogleProj
 import spray.json.JsObject
 
 import scala.concurrent.{ExecutionContext, Future}
-import scala.jdk.CollectionConverters._
 import scala.util.Try
 
 object GoogleServicesDAO {
   def getStorageLogsBucketName(googleProject: GoogleProjectId) = s"storage-logs-${googleProject.value}"
 }
 
-abstract class GoogleServicesDAO(groupsPrefix: String) extends ErrorReportable {
+trait GoogleServicesDAO extends ErrorReportable {
```

Comment:
`groupsPrefix` was unused and this needs to be a trait (interface) so it can be implemented by the disabled service dynamic proxy